Portfolio/index.html

```diff
@@ -6,8 +6,98 @@
     <link rel="stylesheet" href="index.css" />
```

Comment:
The inline styles for the body element could be moved to an external CSS file for better separation of concerns.